sdc/src/util/json.d

```diff
@@ -1993,19 +2015,11 @@ private struct JSONReader(InputRange) {
 
             scope(failure) --column;
 
-            try {
-                inputRange.popFront();
-            } catch {
-                throw complaint("Unexpected end of input");
-            }
+            inputRange.popFront();
```

Comment:
Why catch the exception from the inputRange? Why not just let it bubble through? IIRC a raw catch works on Throwable, so it caught Errors as well, which is bad®.